source/includes/stackpath/_workloads.md

```diff
@@ -292,13 +292,13 @@ Create a new workload in a given [environment](#administration-environments).
 
 Required | &nbsp;
 ------- | -----------
- `name`<br/>*string* | The name of the workload.
+ `name`<br/>*string* | The name of the workload. The string "workloadname-deploymentname" must not exceed 40 characters.
```

Comment:
```suggestion
 `name`<br/>*string* | The name of the workload. The deployment name for a workload must not exceed 40 characters.
```

Reply:
but wouldn't it be confusing for the user? they could have both the strings under 40 chars and the combination might still be invalid (>40)

Reply:
"workloadname-deploymentname" is a bit confusing. How about - 
```suggestion
 `name`<br/>*string* | The name of the workload. The length of `<workload_name>-<deployment_name>` must not exceed 40 characters.
```

Reply:
yes, perfect. I'll update it

Reply:
a5efd65